druid_test.go

```diff
@@ -19,3 +19,17 @@ func TestSetBaseURLWithSuffix(t *testing.T) {
 	assert.Nil(err, "error should be nil")
 	assert.Equal(d.baseURL, wantBaseURL, "they should not be equal")
 }
+
+func TestNewClientWithSkipVerify(t *testing.T) {
+	assert := assert.New(t)
+
+	var druidOpts []ClientOption
+	druidOpts = append(druidOpts, WithSkipTLSVerify(true))
+
+	d, err := NewClient("localhost:8082", druidOpts...)
+	assert.Nil(err, "error should be nil")
+	assert.NotNil(d, "client should not be nil")
+	assert.True(d.skipTLSVerify, "they should not be equal")
```

Comment:
Probably better to check `d.http.HTTPClient.Transport.TLSClientConfig.InsecureSkipVerif` to make sure the configuration got propagated as expected

Reply:
nb: you gonna need that https://github.com/grafadruid/go-druid/pull/38#discussion_r587287718

Reply:
Yep got it in new changes.